assemble/bin/accumulo

```diff
@@ -89,4 +89,21 @@ function main() {
   exec "${JAVA[@]}" "${JAVA_OPTS[@]}" org.apache.accumulo.start.Main "$@"
 }
 
+args=("$@")
+
+if [ -n "$ACCUMULO_BIND_ADDR" ]; then
+  args+=("-o")
+  args+=("general.process.bind.addr=${env:ACCUMULO_BIND_ADDR}")
+fi
+
+if [ -n "$ACCUMULO_COMPACTOR_QUEUE" ]; then
+  args+=("-o")
+  args+=("compactor.queue=${env:ACCUMULO_COMPACTOR_QUEUE}")
+fi
+
+if [ -n "$ACCUMULO_SSERVER_GROUP" ]; then
+  args+=("-o")
+  args+=("sserver.group=${env:ACCUMULO_SSERVER_GROUP}")
+fi
```

Comment:
You can combine these into one line. Bash arrays are space delimited.

However, I don't think this will work in the bash script. The `${env:XXX}` syntax works inside the configuration file. Here in bash, it should just be `$XXX`.

Also, I think we've standardized on the double square braces `[[`, so we're not potentially executing an external process, `/usr/bin/[` (yes, that's an actual process and how POSIX shells process the contents of the square braces; the double brace makes it clear that it's done inside bash as a language feature, not as an external process).

```suggestion
if [[ -n "$ACCUMULO_BIND_ADDR" ]]; then
  args+=("-o" "general.process.bind.addr=$ACCUMULO_BIND_ADDR")
fi

if [[ -n "$ACCUMULO_COMPACTOR_QUEUE" ]]; then
  args+=("-o" "compactor.queue=$ACCUMULO_COMPACTOR_QUEUE")
fi

if [[ -n "$ACCUMULO_SSERVER_GROUP" ]; then
  args+=("-o" "sserver.group=$ACCUMULO_SSERVER_GROUP")
fi
```

Reply:
resolved in later commit